apps/docs/src/app/core/component-docs/wizard/examples/wizard-visible-summary-example.component.html

```diff
@@ -12,6 +12,7 @@ <h2 fd-title headerSize="2">Checkout</h2>
                             [status]="step1status"
                             (statusChange)="statusChanged(1, $event)"
                             label="Product Type"
+                            aria-label="Step 1: Product Type"
```

Comment:
Could you justify the need of aria-label by providing info about the problem and link to a11y documentation? 
Also, I don't think it's reasonable to make the user provide twice the same information. IF aria-label is really necessary you can work on generating it behind the scenes from the label prop and make it an input prop in case the user wants to set it to smth different. But I really want to understand why aria-label is needed first place.